R/predict_response.R

```diff
@@ -633,6 +633,7 @@ predict_response <- function(model,
                              weights = NULL,
                              bias_correction = FALSE,
                              verbose = TRUE,
+                             na.action = "omit", # Default action
```

Comment:
Please check if the argument name follows the convention of other argument names.